### ISSUE TITLE
MinGW makefile

### ISSUE THREAD

**Issue by leilei-** [divVerent/s2tc#11]
Because on Windows, it's a bit ...... difficult to have an automake/autoconf setup going.

**Comment by divVerent** [comment]
Good idea, but what would be using this library on Windows? Or is this about the commandline tools only?

And... would a batch file work too for this special case?

**Comment by leilei-** [comment]
I was hoping to try s2tc_compress on all my data alone just to drop nvcompress from use.

(I did actually try to get Automake/autoconf going with my Msys setup, but it complains about m4 not being there despite actually being there, hence the creation of this issue)

**Comment by divVerent** [comment]
Basically no problem - just need to find some time to run make in some mingw setup and dump the commands it runs.
